package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-fabrica",
-  "version": "0.0.1-alpha",
+  "version": "0.0.1",
```

Comment:
myślę, że powinniśmy mieć jakiś test weryfikujący, czy wersja tutaj i `FABRICA_IMAGE_TAG` to to samo